packages/generator-langium/templates/web/.package.json

```
@@ -4,7 +4,7 @@
         "prepare:public": "shx mkdir -p ./public && shx cp -fr ./src/static/* ./public/",
         "copy:monaco-editor-wrapper": "shx cp -fr ./node_modules/monaco-editor-wrapper/bundle ./public/monaco-editor-wrapper",
         "copy:monaco-workers": "shx cp -fr ./node_modules/monaco-editor-workers/dist/ ./public/monaco-editor-workers",
-        "copy:monarch-syntax": "shx cp -fr ./out/syntaxes/ ./public/syntaxes",
+        "copy:monarch-syntax": "shx cp -fr ./out/syntaxes/ ./public/",
```

It seems like ending the source of `cp` with a `/` is [bad practice](https://stackoverflow.com/questions/18158248/should-i-put-trailing-slash-after-source-and-destination-when-copy-folders), We should use something like this instead, which explains better that the target is `public/syntaxes`: 
```suggestion
        "copy:monarch-syntax": "shx cp -fr ./out/syntaxes ./public/syntaxes",
```

---

Done.